tools/monorepo-utils/src/ci-jobs/lib/__tests__/config.spec.ts

```diff
@@ -192,6 +192,47 @@ describe( 'Config', () => {
 			} );
 		} );
 
+		it( 'should parse test config with report', () => {
+			const parsed = parseCIConfig( {
+				name: 'foo',
+				config: {
+					ci: {
+						tests: [
+							{
+								name: 'default',
+								changes: '/src/**/*.{js,jsx,ts,tsx}',
+								command: 'foo',
+								report: {
+									resultsBlobName: 'foo-blob-report',
+									resultsPath: '/test-results',
+									allure: true,
+								},
+							},
+						],
+					},
+				},
+			} );
+
+			expect( parsed ).toMatchObject( {
+				jobs: [
+					{
+						type: JobType.Test,
+						name: 'default',
+						changes: [
+							/^package\.json$/,
+							makeRe( '/src/**/*.{js,jsx,ts,tsx}' ),
+						],
+						command: 'foo',
```

Comment:
Is `foo` right here?  Or should it be something different?

Reply:
Yes, any string works there for the test purpose.